beta/src/content/learn/separating-events-from-effects.md

```diff
@@ -796,6 +796,22 @@ With `useEvent`, there is no need to "lie" to the linter, and the code works as
 
 <Sandpack>
 
+```json package.json hidden
+{
+  "dependencies": {
+    "react": "experimental",
+    "react-dom": "experimental",
+    "react-scripts": "latest"
```

Comment:
This should be at least pinned to SemVer major e.g. `^4.0.0`. Otherwise this Sandpack may break once a new major version is released.

Reply:
`latest` is used in other sandpacks as well?

Reply:
> `latest` is used in other sandpacks as well?

Yes!